src/System.Buffers.Experimental/System/Buffers/SpanWriter.cs

```diff
@@ -0,0 +1,207 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Move SpanWriter.cs to System.Binary?
https://github.com/dotnet/corefxlab/blob/master/src/System.Binary/System/Binary/SpanWriter.cs

Reply:
It's not binary. It mainly writes text for now. Besides, for now I want to keep it in experimental. I am not sure this whole idea of passing in transformations is sound.